### DISCUSSION TITLE
Weblate: open source web-based translation tool with version control

### DISCUSSION THREAD

**Discussion by NamelessGO** [dynobo/normcap#630] [Ideas]
[Weblate](https://weblate.org/) is libre software web-based continuous localization system, used by over 2500 libre projects and companies in more than 165 countries

10 Benefits from [Features](https://weblate.org/en/features/):
- [Continuous Localization](https://docs.weblate.org/en/latest/admin/continuous.html#continuous-translation): Automated localization workflow follows your project development closely.
- [Quality Checks](https://docs.weblate.org/en/latest/user/checks.html): Customizable quality checks helps improve translation quality.
- [Smart Notifications](https://docs.weblate.org/en/latest/user/profile.html#subscriptions): Translators can be notified of new strings to translate or other events.
- [Translation Context](https://docs.weblate.org/en/latest/admin/translating.html#additional): Description, nearby strings, images, screenshots related source code and glossary entries—for every string.
- [Easy Reviews](https://docs.weblate.org/en/latest/workflows.html): Several options for translators to review contributions made by peers.
- [Translation memory](https://docs.weblate.org/en/latest/admin/memory.html): Reuse existing translations to save time and costs.
- [Automatic translation](https://docs.weblate.org/en/latest/admin/machine.html): Translations or suggestions can be done by machine translation services or AI.
- [Promotion](https://docs.weblate.org/en/latest/devel/share.html): Built-in features to attract a wider audience for your translation project.
- [Translation add-ons](https://docs.weblate.org/en/latest/admin/addons.html): Customize localization workflow with various add-ons.
- [Glossaries](https://docs.weblate.org/en/latest/user/translating.html#glossary): Define frequently used terminology for consistency.

References

- [Features](https://weblate.org/en/features/)
- [Docs](https://docs.weblate.org/en/latest/)
- [Projects hosted by Weblate](https://hosted.weblate.org/) (Excluded Self Hosted projects)
- [Source Code](https://github.com/WeblateOrg/weblate)
- [Wiki](https://www.wikiwand.com/en/Weblate)

<img width="1027" alt="image" src="https://github.com/dynobo/normcap/assets/66227691/c98c934f-e0a0-44a1-9c04-913a0c5e72fd">

<img width="402" alt="image" src="https://github.com/dynobo/normcap/assets/66227691/0331d82a-5c2f-4743-bb9b-2023f14973f2">

<img width="780" alt="image" src="https://github.com/dynobo/normcap/assets/66227691/a2c6d113-a415-463a-a757-e00803e86307">

[Hosting](https://weblate.org/en/hosting/): If you feel like being supported by Weblate will help you, [set up your libre project](https://hosted.weblate.org/hosting/) and get the Libre plan gratis. It has the same limits as the Advanced plan, but only for public projects.

**Comment by dynobo** [comment]
Thanks for the suggestion!! I just create an account right now and I'm waiting for the approval for the free "libre" plan. Once this is done, I'll adjust the documentation on how to contribute to translations. :slightly_smiling_face: :+1:
